Tools/CISupport/ews-build/factories_unittest.py

```diff
@@ -349,6 +349,26 @@ class TestExpectedBuildSteps(unittest.TestCase):
             'validate-change',
             'compile-webkit'
         ],
+        'WPE-WK2-Tests-EWS': [
```

Comment:
We need `Tools/CISupport/ews-app` changes too, I think, is that right @aj062?

Reply:
Sure. I'm fine with having the bots running a few days on the builbot instance to ensure they run stable and as expected before adding the checkboxes to github PRs.

Reply:
ping?
I would like to get this reviewed and landed, it has been here for 1 month already waiting and the new machines are mostly idle waiting to start running the tests.